tests/store.spec.js

```diff
@@ -1323,6 +1343,26 @@ describe('API store', () => {
         expect(axiosMock.history.get.length).toEqual(3)
       })
 
+      it('does not delete object from the store if api fails', async () => {
```

Comment:
why are there new tests in this "Move to vitest" PR?

Reply:
Because it reported the coverage not exactly the same as before with jest.